packages/client/hmi-client/src/components/asset/tera-asset.vue

```diff
@@ -13,14 +13,17 @@
 				/>
 			</aside>
 		</header>
-		<header id="asset-top" ref="headerRef">
-			<template v-if="!hideHeader">
+		<template v-if="!hideHeader">
+			<header id="asset-top" ref="headerRef">
 				<section>
 					<!-- put the buttons above the title if there is an overline -->
 					<div v-if="overline" class="vertically-center">
 						<span class="overline">{{ overline }}</span>
 						<slot name="edit-buttons" />
 					</div>
+					<div>
+						<slot name="info-bar" />
+					</div>
```

Comment:
Does it need to be inside a `<div>`?

Reply:
@YohannParis  you're right this isn't needed removed